src/emulator/host/src/app.cpp

```diff
@@ -28,9 +28,9 @@
 #include <kernel/state.h>
 #include <kernel/thread/thread_functions.h>
 #include <util/find.h>
-#include <util/fs.h>
 #include <util/log.h>
 #include <util/string_convert.h>
+#include <boost/filesystem.hpp>
```

Comment:
nit: includes from dependencies should go below this line (along with `SDL.h` and `gl.h` below)